hpi-swt2/lecture-portal
# Restyle start and stop button in polls for the lecture view

# Resolves #295 

This PR changes the start and stop button in polls for the lecture view.

Acceptance Criteria:
    - The "Start/Stop" Button is a primary button
    - The "Start/Stop" Button is centered beneath the title (not at the bottom of the page
    - The Back-Button is relabeled to "Show all Polls" (But still has that fancy arrow)


Testing steps:
- open the polls view as a lecturer and check the stop/start button.

---

Definition Of Done:
